config/routes.rb

```diff
@@ -1,3 +1,18 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/merchants', to: 'merchants#index'
+  get '/merchants/new', to: 'merchants#new'
+  get '/merchants/:id', to: 'merchants#show'
+  post '/merchants', to: 'merchants#create'
+  get '/merchants/:id/update', to: 'merchants#edit'
```

Comment:
this route works just fine, but the conventional way to make this edit route would be to use the word edit in the path name instead of update 
'/merchants/:id/edit